docs/apps/sofi3d.md

```diff
@@ -0,0 +1,240 @@
+---
+tags:
+  - Free
+---
+
+# SOFI3D
+
+[SOFI3D](https://gitlab.kit.edu/kit/gpi/ag/software/sofi3d) is an application for 3D Finite-Difference Seismic Wave Simulation.
+
+## Available
+
+These tools are available on Puhti:
+
+* partmodel
+* seismerge
+* snapmerge
+* sofi3D
+* sofi3D_acoustic
+
+## Usage
+
+The SOFI3D module is available on Puhti and can be loaded using the following command:
+
+`module load sofi3d`
+
+The application can be run with:
+
+`sofi3D`
+
+## Example batch job
+
+SOFI3D creates many auxiliary files, so in this example subdirectories are used to reduce clutter.
+
+```bash
+#!/bin/bash
+#SBATCH --account=<your-project-unix-id>
```

Comment:
```suggestion
#SBATCH --account=<project>
```
I'd update the mentions of your-project-unix-id to what is used in eg the batch job example page, which is <project>. And add a note that that needs to be changed in the script before running with an hint on the format,which is project_200xxxx